packages/@react-stately/virtualizer/src/LayoutInfo.ts

```diff
@@ -36,6 +36,11 @@ export class LayoutInfo {
    */
   parentKey: Key | null;
 
+  /** 
+   * Content for this view if it was generated by the layout rather than coming from the Collection.
+   */
+  content: any | null;
```

Comment:
Ideally we'd use a generic here but that would require a lot of changes to propagate everywhere...

Reply:
lot of changes, but would they be breaking?